.github/workflows/build-test.yml

```diff
@@ -220,67 +220,6 @@ jobs:
           skip-label: 'skip telemetry'
           failure-message: 'TELEMETRY.md should be updated when adding or removing telemetry. Please run "npm run generateTelemetry"'
 
-  vsc_api_check:
```

Comment:
The proposed API d.ts files are always downloaded during compilation, hence its never possibel to have outdated items.
Also, the proposed API d.ts files aren't commited into the repo anymore.